conf/sample.ini

```diff
@@ -1089,6 +1089,10 @@
 # Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server,
 # which this setting can help protect against by only allowing a certain amount of concurrent requests.
 ;concurrent_render_request_limit = 30
+# RenderKey max duration expressed in duration (`10s`, `1m`)
+# The renderkey is used by the image renderer to access grafana (it's like an API key). The default validity is 5 minutes. This should be more than enough for most deployments.
+# Change the value only if image rendering is failing and you see `Failed to get the render key from cache` in grafana logs.
```

Comment:
This should match the description on `defaults.ini`, please 🙏🏻

Reply:
Of course ! I've gone too quickly and I don't know how I missed that ! Sorry